### ISSUE TITLE
User's profile image is not saved in a database. 

### ISSUE THREAD

**Issue by nikhil-sachdeva** [openMF/mobile-wallet#365]
**Summary:**

UI should be saved locally for the user.

**Steps to reproduce:**

Upload user image in edit profile tab.

**Expected behavior:**

Locally saving image will lead to just retrieving the save location and not selecting the image again.
Even the removeProfileImage() method can be applied.
~~~ java
    @Override
    public void removeProfileImage() {
        // TODO: Remove image from database
    }
~~~

**Observed behavior:**

I didn't see any local storage of user's profile image.

**Device and Android version:**

Redmi Note 3

**Comment by shaliniv16** [comment]
@nikhil-sachdeva This issue is similar to #214

**Comment by nikhil-sachdeva** [comment]
@shaliniv16 This issue is to implement local storage of the image uploaded. If the API does not support image upload in profile data, the app must save it locally to fulfil the need.

**Comment by Vipinsoni38** [comment]
Exactly @nikhil-sachdeva !
are you working on it or shall i proceed with it

**Comment by Vipinsoni38** [comment]
also once this is done then we can extract user profile by Google Sign in which would be more effective

**Comment by nikhil-sachdeva** [comment]
@Vipinsoni38 Yes, I'm working on it, once the serves is up again, I'll be sending a PR. 
@luckyman20 @naman14 @ankurs287 Any inputs regarding the approach for this issue?

**Comment by Vipinsoni38** [comment]
@nikhil-sachdeva i did resolve the issue yesterday

saved the image in device as bitmap and again extracted it

**Comment by devansh-299** [comment]
@luckyman20 can I work on this issue as this problem hasn't been addressed from a long time?

**Comment by luckyman20** [comment]
@nikhil-sachdeva This method can pose possible security concerns. We should have an API for uploading the image.

**Comment by therajanmaurya** [comment]
Fixed in compose implementation
